docs/tools/device-provisioning.md

```diff
@@ -257,6 +257,48 @@ The query verb supports the following parameters:
 |--help|no|Display this help screen.|
 |--version|no|Display version information.|
 
+### rotate-certificate
```

Comment:
also add these two command to the table on line 63.
(there is also a small spelling mistake on line 65 (`Lits` instead of `List`), but I cannot comment on those lines from here.